Cargo.toml

```diff
@@ -35,7 +35,7 @@ spin = "0.5"
 reqwest = { version = "0.9.5", optional = true }
 
 [target.'cfg(target_os = "linux")'.dependencies]
-procinfo = { version = "0.4", optional = true }
+procinfo = { git = "https://github.com/tikv/procinfo-rs", optional = true }
```

Comment:
maybe not appropriate to use pingcap's own procinfo?

Reply:
Why not? We use it in TiKV

Reply:
But `tikv/procinfo-rs` is not published as a crate, right? Which would make this crate also not publishable anymore.
I do personally have projects (already on crates.io) which need to consume this from the registry.

Have you maybe considered switching to https://docs.rs/procfs/0.5.3/procfs/ or checking with @danburkert if they are open to co-maintainers? (does PingCAP want to maintain that crate?)

Reply:
I can change this back if we think it is really important, the key thing is that we move off 0.3 which the version of prometheus in TiKV is currently using